### PR TITLE
fix: back link non functional on event keywords

### PR DESCRIPTION
**Pull request by PekkaKela**
### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr247.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr247.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr247.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr247.dev.hel.ninja :laughing::tada::tada::tada:

File: apps/sports-helsinki/src/domain/venue/venueKeywords/__tests__/VenueKeywords.test.tsx [hunk 1]
```diff
@@ -33,7 +33,9 @@ it('should render keywords and handle click', async () => {
     screen.getByRole('link', { name: new RegExp(keywordNames[0], 'i') })
   );
   expect(router).toMatchObject({
-    asPath: `/haku?q=${encodeURIComponent(capitalize(keywordNames[0]))}`,
+    asPath: `/haku?q=${encodeURIComponent(
+      capitalize(keywordNames[0])
+    )}&searchType=Venue`,
```
Comment (kristianjokela) [inline apps/sports-helsinki/src/domain/venue/venueKeywords/__tests__/VenueKeywords.test.tsx:38]: Maybe would be safer to construct this test like the actual query, ie. with the `PARAM_SEARCH_TYPE` constant.

**Review by nikomakela — CHANGES_REQUESTED** [review]
The event / course details pages still needs the same feature

File: apps/sports-helsinki/src/domain/venue/venueKeywords/__tests__/VenueKeywords.test.tsx (hunk 1 above)
Comment (PekkaKela) [reply apps/sports-helsinki/src/domain/venue/venueKeywords/__tests__/VenueKeywords.test.tsx]: Changed this to contruct the query same way as it does in VenueKeywords

File: apps/sports-helsinki/src/domain/venue/venueKeywords/VenueKeywords.tsx [hunk 2]
```diff
@@ -60,6 +61,7 @@ const VenueKeywords: React.FC<Props> = ({
     () => {
       const search = buildQueryFromObject({
         q: type === 'text' ? [value] : [],
+        [PARAM_SEARCH_TYPE]: 'Venue',
```
[the remark this answers is not in the document]
Comment (nikomakela) [reply apps/sports-helsinki/src/domain/venue/venueKeywords/VenueKeywords.tsx:64]: Adding the new search parameters to the UI or to the query might be a bit bigger task, but like I said in previous comment, it's something that we must do at some point anyway, or we need to disable the whole link feature.

[the remark this answers is not in the document]
Comment (PekkaKela) [reply apps/sports-helsinki/src/domain/venue/venueKeywords/VenueKeywords.tsx:64]: Will do these changes to event and course pages aswell

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr247.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr247.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr247.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr247.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr247.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr247.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr247.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr247.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr247.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by PekkaKela** [comment]
Changed PR to draft as this PR is waiting for LIIKUNTA-425 to be started.

**Comment by karisal-anders** [comment]
Closed to clean up the open pull requests list.

**Pull request closed without merging**